cmd/tracee-rules/output.go

```diff
@@ -72,6 +80,11 @@ func setupOutput(w io.Writer, webhook string, webhookTemplate string, contentTyp
 					log.Printf("error sending to webhook: %v", err)
 				}
 			}
+			if fluentForwardUrl != "" {
```

Comment:
if fluentForwardUrl != "" && tFluentForward != nil {}

Reply:
Again, the code is following the current convention and a null template is caught later with a specific error: you may specify a URL without a template for example which is a more specific error than a silent not sending.